docs/DEVELOPING.md

```diff
@@ -343,6 +343,114 @@ $ make ftest NAME=azure_blob_storage
 
 ℹ️ The e2e test uses default values defined in [configure Azure Blob Storage connector](#configure-azure-blob-storage-connector)
 
+
+## Microsoft SQL Connector
+
+The [Elastic Microsoft SQL connector](https://github.com/elastic/connectors-python/blob/8.7/connectors/sources/mssql.py) is provided in the Elastic connectors python framework and can be used via [build a connector](https://www.elastic.co/guide/en/enterprise-search/current/build-connector.html).
+
+### Availability and prerequisites
+
+⚠️ _Currently, this connector is available in **beta** starting in 8.7_.
+Features in beta are subject to change and are not covered by the service level agreement (SLA) of features that have reached general availability (GA).
+
+Elastic versions 8.6.0+ are compatible with Elastic connector frameworks. Your deployment must include the Elasticsearch, Kibana, and Enterprise Search services.
+
+**Prerequisite**
+
+The Microsoft SQL connector requires an ODBC driver to execute. To install ODBC driver please refer following [documentation](https://learn.microsoft.com/en-us/sql/connect/odbc/linux-mac/installing-the-driver-manager).
+
+### Setup and basic usage
+
+Complete the following steps to deploy the connector:
+
+1. [Gather Microsoft SQL details](#gather-microsoft-sql-details)
+2. [Configure Microsoft SQL connector](#configure-microsoft-sql-connector)
+
+#### Gather Microsoft SQL details
+
+Collect the information that is required to connect to your Microsoft SQL Server:
+
+- The server host address where the Microsoft SQL Server is hosted.
+- The port where the Microsoft SQL Server is hosted.
+- The username the connector will use to log in to the Microsoft SQL Server.
+- The password the connector will use to log in to the Microsoft SQL Server.
+- The database name where the connector will query data.
+
+#### Configure Microsoft SQL connector
+
+The following configuration fields need to be provided for setting up the connector:
+
+##### `host`
+
+The server host address where the Microsoft SQL Server is hosted. Default value is `127.0.0.1`. Examples:
+
+  - `192.158.1.38`
+  - `demo.instance.demo-region.demo.service.com`
+
+##### `port`
+
+The port where the Microsoft SQL Server is hosted. Default value is `9090`. Examples:
+
+  - `5432`
+  - `9090`
+
+##### `username`
+
+The username of the account for Microsoft SQL Server. Default value is `admin`.
+
+##### `password`
+
+The password of the account to be used for the Microsoft SQL Server. Default value is `Password_123`.
+
+##### `database`
+
+Name of the Microsoft SQL Server database. Default value is `xe`. Examples:
```

Comment:
This is a copy and paste error.  There's no default `xe` SQL Sever database, that's the default for Oracle Express Edition 😄 My 2 cents is to have no default for this field but make it required to be set.

Reply:
The rule of thumb should be to set default values where it makes sense, otherwise we should leave the fields empty. 

We're currently working on providing better guidance on configurable fields such as adding support for validations, tooltips or placeholders where it makes sense starting with 8.8.

Reply:
Based on this [comment](https://github.com/elastic/connectors-python/pull/549/files#r1127785532) in the other feedback, it seems like the default value set to `xe` is required only because of E2E testing.  Perhaps we can revisit in 8.8 to see if we can decouple E2E testing dependency on requiring a default value to be set for the database field for all database connectors?